cumulus/ansible/README.md

```diff
@@ -22,17 +22,20 @@
 
 ```json
     {"name": "test cluster",
-    "type": "ansible",
-	"playbook": "sge",
-	"cluster_config": {
-         "master_instance_type": "t2.micro",
-         "master_instance_ami": "ami-03de3c63",
-         "node_instance_count": 2,
-         "node_instance_type": "t2.micro",
-         "node_instance_ami": "ami-03de3c63",
-         "ansible_ssh_user": "ubuntu"
+     "type": "ansible",
+     "config": {
+       "launch": {
+         "spec": "sge",
```

Comment:
spec should really be ```ec2``` -  why don't we go ahead and rename the sge.yml to ec2.yml and either drop default.yml or symlink it to sge.yml?